packages/icon-builder/package.json

```diff
@@ -22,14 +22,16 @@
     "react-component"
   ],
   "scripts": {
-    "prebuild": "rimraf build",
-    "build": "npm run build:icons && npm run build:copy-files",
-    "build:icons": "node ./build.js --output-dir ./build --svg-dir ./node_modules/material-design-icons --glob '/**/production/*_24px.svg' --renameFilter ./filters/rename/material-design-icons.js",
+    "prebuild": "rimraf build/*.js",
+    "build": "npm run build:icons && npm run build:babel && npm run build:copy-files",
+    "build:babel": "babel --presets es2015 -d ./build ./src",
+    "build:icons": "node ./build.js --output-dir ./src --svg-dir ./node_modules/material-design-icons --glob '/**/production/*_24px.svg' --renameFilter ./filters/rename/material-design-icons.js",
     "build:copy-files": "babel-node ./copy-files.js",
     "test": "grunt"
   },
   "devDependencies": {
-    "babel-cli": "^6.0.0",
+    "babel-cli": "^6.24.0",
+    "babel-preset-es2015": "^6.24.0",
```

Comment:
Oops! - I guess it was picking up `.babelrc` from the parent directory. I've made it explicit.